src/Domain/DomainHelpers.hpp

```diff
@@ -72,3 +72,38 @@ std::vector<std::unique_ptr<CoordinateMapBase<Frame::Logical, TargetFrame, 3>>>
 wedge_coordinate_maps(double inner_radius, double outer_radius,
                       double inner_sphericity, double outer_sphericity,
                       bool use_equiangular_map) noexcept;
+
+/// \ingroup ComputationalDomainGroup
+/// Generates the corners for a Domain which is made of one or more layers
+/// of Blocks fully enveloping an interior volume, e.g. Shell or Sphere. The
+/// `number_of_layers` specifies how many of these layers of blocks to have
+/// in the final domain.
+/// `include_central_block` is set to `true` in Sphere and `false` in Shell.
+/// The `central_block_corners` are used to seed the values for the corners
+/// of the surrounding blocks.
+std::vector<std::array<size_t, 8>> corners_for_radially_layered_domains(
+    size_t number_of_layers, bool include_central_block,
+    const std::array<size_t, 8>& central_block_corners = {
+        {1, 2, 3, 4, 5, 6, 7, 8}}) noexcept;
+
+/// \ingroup ComputationalDomainGroup
+/// Generates the corners for a BBH-like Domain which is made of one or more
+/// layers of Blocks surrounding two interior blocks. The
+/// `number_of_radial_layers` gives the number of layers that fully envelop
+/// each interior block with six blocks each. The `number_of_biradial_layers`
+/// gives the number of layers that fully cover both blocks at once, using ten
+/// blocks per layer as opposed to six.
+std::vector<std::array<size_t, 8>> corners_for_biradially_layered_domains(
+    size_t number_of_radial_layers, size_t number_of_biradial_layers,
+    bool include_central_block_lhs, bool include_central_block_rhs,
+    const std::array<size_t, 8>& central_block_corners_lhs = {
+        {1, 2, 3, 4, 5, 6, 7, 8}}) noexcept;
+
+/// \ingroup ComputationalDomainGroup
+/// Generates the corners for a Domain which is made of one or more layers
+/// of Blocks surrounding an interior volume, e.g. Shell or Sphere. The
+/// `number_of_partial_layers` parameter is used when constructing a BBH-like
+/// Domain where there are layers of blocks which surround the interior volume
+/// using only 5 blocks as opposed to 6.
```

Comment:
It is not very clear what this is doing, or how this is different from `corners_for_radially_layered_domains`. What is the number 5 coming from?

Reply:
This was the original function that was factored into the two functions for radial and biradially layered domains - I left it here so it could still be compared for the other two - apologies for the confusion